proto/autokitteh/triggers/v1/trigger.proto

```diff
@@ -9,7 +9,7 @@ import "buf/validate/validate.proto";
 message Trigger {
   string trigger_id = 1;
   string name = 2;
-  string connection_id = 3; // could be empty
+  string connection_id = 3;
```

Comment:
This is essential for the current implementation of scheduler triggers - please sync with @ig-ra

Reply:
ditto.